apps/central/src/components/Groups.vue

```diff
@@ -115,17 +115,18 @@ export default {
       return this.schemasFiltered.length;
     },
     schemasFiltered() {
```

Comment:
"schemasFiltered" now more a of a "schemasFilteredAndSorted'

Reply:
done